guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+title: Creating a new a wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
```

Comment:
Are you planning on a preview image? Doesn't have to be an image that appears on the page, can be just for the link preview when sharing.

Reply:
Yeah, do you think we need a banner image as well?

Reply:
Overall, this is your decision. My call would be to say yes. "A picture is worth a thousand words" is not an exaggeration, and a good header image can communicate way more quickly and efficiently what the page is about and set the tone than a 1-2 line text title. So I would heavily invest in creating good visuals.

Reply:
Spoken a bit about this on slack already but there is interest from other community members to take this on so unlikely to be a part of this PR.

Reply:
Made an issue for this so people know its needed, I couldn't find the slack discussion what channel was it in Conor?

Reply:
I can't remember the thread now but we have similar issue open here as well - https://github.com/BitcoinDesign/Guide/issues/207